public/favicon.svg

```diff
@@ -0,0 +1,3 @@
+<svg xmlns="http://www.w3.org/2000/svg">
+  <text y="32" font-size="32">🪑</text>
```

Comment:
Its a suggestion. you can try with different favicon, as it symbolizes only chair which is part of furniture.

Reply:
okay Thanks